### PR TITLE
Changes to FPGA tutorial shannonization to make it more like other tutorials.

### PR DESCRIPTION
**Pull request by tyoungsc**
# Existing Sample Changes
## Description
Merged the two vcxproj files into one (like all other tutorials), changed one of the A10 II targets, and fixed indenting of a comment block.

### CODE REVIEW

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp [hunk 1]
```diff
@@ -268,28 +268,28 @@ int main(int argc, char** argv) {
...
-  // can only achieve an II of 3 while versions 1 and 2 can only achieve
-  // an II of 2. On Stratix 10 and Agilex, we can achieve an II of 1 if we use
-  // non-blocking pipe reads in the IntersectionKernel, which is shown in
-  // version 3 of the kernel.
-  //
+    // Instantiate multiple versions of the kernel
+    // The II achieved by the compiler can differ between FPGA architectures
+    //
+    // On Arria 10, we are able to achieve an II of 1 for versions 1 and 2 of
+    // the kernel (not version 0).
```
Comment (mdbtucker) [inline DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp:275]: Is this comment still accurate?  If I undersand the code change you made below it seems like you're now asking for an II of 1 on version 0 of the kernel?

**Review by mdbtucker — APPROVED** [review]
Approving to trigger CI

File: DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp (hunk 1 above)
Comment (tyoungsc) [reply DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/shannonization/src/shannonization.cpp]: Fixed!

**Review by mdbtucker — APPROVED** [review]
Looks good to me, we'll merge after the 2021.3 tag is created

**Comment by mdbtucker** [comment]
@tyoungsc the CI failed in Windows because of a long path name.  Is there anything we can do about that?

**Comment by tyoungsc** [comment]
This PR is intended for 2021.3. Can it be relabeled and merged in?
W.r.t to long path: not really... make the filenames shorter either via the compiler or a shorter root dir.

**Review by mdbtucker — APPROVED** [review]

**Comment by tyoungsc** [comment]
@mdbtucker can this be merged in now?

**Pull request merged**